db/seeds.rb

```diff
@@ -12,6 +12,16 @@
   puts 'ユーザーの初期データインポートに成功しました。'
 end
 
+EMAIL = 'admin@example.com'
+PASSWORD = 'password'
+PASSWORD_CONFIRMATION = 'password'
+
+#管理者が存在しない時だけ新規作成 
+AdminUser.find_or_create_by!(email: EMAIL) do |user|
+   user.password = PASSWORD
+   user.password = PASSWORD_CONFIRMATION
+   puts '管理者の初期データインポートに成功しました。'
+
```

Comment:
deviseの `confirmable` は入れていませんので，passwordのみでOKです。

```suggestion
AdminUser.find_or_create_by!(email: ADMIN_EMAIL) do |admin_user|
   admin_user.password = PASSWORD
   puts '管理者の初期データインポートに成功しました。'
end
```